GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -619,7 +619,7 @@ private void CheckNotInsideExistingRepo(string normalizedEnlistmentRootPath)
 
             try
             {
-                RepoMetadata.Instance.SaveCloneMetadata(tracer, enlistment);
+                RepoMetadata.Instance.SaveCloneMetadata(tracer, enlistment, GVFSPlatform.Instance.DiskLayout.CurrentMajorVersion, GVFSPlatform.Instance.DiskLayout.CurrentMinorVersion);

```

Comment:
I don't think it hurts anything, but what was the reason for making the change here rather than using `GVFSPlatform.Instance.DiskLayout` directly in `RepoMetdata`?

Reply:
Good point I should do the change directly in RepoMetaData